simulator/internal/internal_cgo.go

```diff
@@ -19,6 +19,8 @@ package internal
 // // Silence known warnings from the reference code and CGO code.
 // #cgo CFLAGS: -Wno-missing-braces -Wno-empty-body -Wno-unused-variable -Wno-uninitialized
 // // Link against the system OpenSSL
+// // Silence openssl deprecation warnings for ms-tpm-20-ref
+// #cgo CFLAGS: -Wno-deprecated-declarations
```

Comment:
This should go above the comment about linking against openssl

Reply:
done